packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -150,8 +146,12 @@ export default async function adjustUserCount(
   const r = await getRethink()
   const orgIds = Array.isArray(orgInput) ? orgInput : [orgInput]
 
+  const user = await getUserById(userId)
```

Comment:
11 I think we can safely use the dataloader here. While we might mutate the `OrganizationUser` we don't touch the `User` object here or in the mutations, so there is no risk of reading stale data.